app/controllers/openid_application_controller.rb

```diff
@@ -111,21 +110,35 @@ def derive_loa(profile)
     end
   end
 
-  def user_from_identity(user_identity, ttl)
-    user = User.new(user_identity.attributes)
+  def token_identifiers
+    # Here the `sub` field is the same value as the `uuid` field from the original upstream ID.me
+    # SAML response. We use this as the primary identifier of the user because, despite openid user
+    # records being controlled by okta, we want to remain consistent with the va.gov SSO process
+    # that consumes the SAML response directly, outside the openid flow.
+    # Example of an upstream uuid for the user: cfa32244569841a090ad9d2f0524cf38
+    # Example of an okta uid for the user: 00u2p9far4ihDAEX82p7
+    @token_identifiers ||= OpenStruct.new(
+      uuid: token_payload['sub'],
+      okta_uid: token_payload['uid']
+    )
+  end
```

Comment:
This particular class exists only to avoid further mistakes in this controller re: uid vs uuid. There's nothing about these names that are endemic to the domain.